server/server.go

```diff
@@ -159,25 +155,26 @@ func (svr *Server) Health(w http.ResponseWriter, _ *http.Request) error {
 func (svr *Server) HandleGet(w http.ResponseWriter, r *http.Request) error {
 	ct, err := ReadCommitmentMode(r)
```

Comment:
knit and irrelevant to PR - ct implies `CommitmentType` but it should probably shorthand notation `cm`

Reply:
[624c815](https://github.com/Layr-Labs/eigenda-proxy/pull/124/commits/624c815c35812c321596d21c0e1fee71f4e40d0c)